regexes/device/mobiles.yml

```diff
@@ -12592,7 +12790,7 @@ Xiaomi:
     - regex: ' MiTV-MSSP1[);/ ]'
       model: 'MiTV 4S'
       device: 'tv'
-    - regex: ' MiTV-AXSO0[);/ ]'
+    - regex: '( MiTV-AXSO0|MiTV4A)[);/ ]'
```

Comment:
Just out of curiosity: Is the space in front of `MiTV` required for this and the other regexes? Guess we could maybe simply remove it...

Reply:
I think not